include/vinecopulib/vinecop/implementation/class.ipp

```diff
@@ -78,6 +78,7 @@ inline Vinecop::Vinecop(const std::vector<std::vector<Bicop>> &pair_copulas,
     check_pair_copulas_rvine_structure(pair_copulas);
 
     pair_copulas_ = pair_copulas;
+    vine_struct_.truncate(pair_copulas.size());
```

Comment:
Is it necessary?

Reply:
Hm probably not necessary, but I think it's a good idea to keep things consistent. We don't really know how a user tries to obtain the truncation level. If we don't do this, we will give two different answers for `get_all_pair_copulas().size()` and `get_rvine_structure().get_trunc_lvl()`.

Reply:
OK